src/Console/WarmupCommand.php

```diff
@@ -6,19 +6,31 @@
 use Jderusse\Warmup\ClassmapReader\ChainReader;
 use Jderusse\Warmup\ClassmapReader\DirectoryReader;
 use Jderusse\Warmup\ClassmapReader\OptimizedReader;
+use Jderusse\Warmup\Compiler\CliCompiler;
 use Jderusse\Warmup\Compiler\PhpServerCompiler;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class WarmupCommand extends BaseCommand
 {
+    const COMPILER_PHP_SERVER = 'php_server';
```

Comment:
I'd rather use `sandbox` as the use of php server is a technical detail of the current implementation